libs/payments/stripe/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "forceConsistentCasingInFileNames": true,
     "strict": true,
     "noImplicitOverride": true,
-    "noPropertyAccessFromIndexSignature": true,
+    "noPropertyAccessFromIndexSignature": false,
```

Comment:
When set to `true`, I ran into the errors below with running tests. I tried to create a `env.d.ts` file, but it didn't help.

```
  libs/shared/log/src/lib/logging.ts:13:17 - error TS4111: Property 'NODE_ENV' comes from an index signature, so it must be accessed with ['NODE_ENV'].

    13 if (process.env.NODE_ENV === 'production') {
                       ~~~~~~~~
    libs/shared/log/src/lib/logging.ts:14:26 - error TS4111: Property 'LOG_LEVEL' comes from an index signature, so it must be accessed with ['LOG_LEVEL'].

    14   logLevel = process.env.LOG_LEVEL || 'info';
                                ~~~~~~~~~
    libs/shared/log/src/lib/logging.ts:26:17 - error TS4111: Property 'WINSTON_LOGGING' comes from an index signature, so it must be accessed with ['WINSTON_LOGGING'].

    26 if (process.env.WINSTON_LOGGING === 'true') {
```

Reply:
I believe these should be accessed via the array brackets as the warning mentions. `process.env["NODE_ENV"]` rather than changing the config here